### ISSUE TITLE
time-input element has a buggy arrow key navigation.

### ISSUE THREAD

**Issue by diregraph** [fooloomanzoo/datetime-picker#20]
When arrow keys are used to navigate in the time-input element, it goes up to seconds and goes back to hours instead of milliseconds. When at hours and the left arrow is pressed it goes to milliseconds. This is because of the following code at `TimeInputPattern`
```
static get inputTemplate() {
        return `
          ${super.inputTemplate || ''}
          <div hidden$="[[_hideHours]]" style="order:7;">
            <number-input hidden$="[[hour12]]" index="7" class="time" min="0" max="23" pad-length="2" value="{{hours}}" placeholder="00"></number-input>
            <number-input hidden$="[[!hour12]]" index="7" class="time" min="1" max="12" pad-length="2" value="{{hours12}}" placeholder="00"></number-input>
          </div>
          <span hidden$="[[_hideMinutes]]" style="order:8;">[[_localTimeSeparator]]</span>
          <number-input hidden$="[[_hideMinutes]]" style="order:9;" index="9" pad-length="2" min="0" max="59" disabled="[[disabled]]" value="{{minutes}}" placeholder="00"></number-input>
          <span hidden$="[[_hideSeconds]]" style="order:10;">[[_localTimeSeparator]]</span>
          <number-input hidden$="[[_hideSeconds]]" style="order:10;" index="11" pad-length="2" min="0" max="59" disabled="[[disabled]]" value="{{seconds}}" placeholder="00"></number-input>
          <span hidden$="[[_hideMilliseconds]]" style="order:11;">[[_localDecimalSeparator]]</span>
          <number-input hidden$="[[_hideMilliseconds]]" style="order:12;" index="12" pad-length="3" min="0" max="999" disabled="[[disabled]]" value="{{milliseconds}}" placeholder="000"></number-input>
          <div class="am-switch" hidden$="[[!hour12]]" style="order:13;" on-mouseup="_switchAm">
            <span hidden$="[[!isAm]]">[[_am]]</span>
            <span hidden$="[[isAm]]">[[_pm]]</span>
          </div>
        `;
      }
```
in the `number-input` for milliseconds it is given 12 for the index . The step for `_focusNumberInput` method at `datetime-input-mixin` is 2. This makes the right arrow key event to skip milliseconds.
Correcting the index to 13 solves the issue.

**Comment by fooloomanzoo** [comment]
You are right, I change that. If you wonder why it is so strange that I use index and order like that, it is because of the ´local-dateformat´. Thanks

**Comment by fooloomanzoo** [comment]
should be fixed in [2.5.1](https://github.com/fooloomanzoo/datetime-picker/releases/tag/2.5.1)

**Comment by diregraph** [comment]
Glad I could help @fooloomanzoo. Thank you for the fix.
